move_base/src/move_base.cpp

```diff
@@ -989,6 +976,22 @@ namespace move_base {
       publishZeroVelocity();
       return false;
     }
+
```

Comment:
Was there a reason to move this block down?  It seems like it would work in either location?  You don't need to change it back, I'm just curious about the move.

Reply:
I initially moved the block down into the if(newPlan) block, when I moved it back I put it in a different place by accident